pkg/utils/file.go

```diff
@@ -0,0 +1,32 @@
+package utils
```

Comment:
I was think it should go into its own package, as that would avoid cyrcular dependency between `pkg/utils` and `pkg/utils/kubeconfig`, as `pkg/utils`  already depends on `pkg/utils/kubeconfig`...